polygon-digital-carbon/src/utils/CarbonProject.ts

```diff
@@ -18,64 +17,20 @@ export function loadOrCreateCarbonProject(registry: string, projectID: string):
     project.country = ''
     project.region = ''
 
-    // Set known values for Verra projects
-    if (registry == 'VERRA') {
-      for (let i = 0; i < VERRA_PROJECT_NAMES.length; i++) {
-        if (projectID == VERRA_PROJECT_NAMES[i][0]) {
-          project.name = VERRA_PROJECT_NAMES[i][1]
-          project.country = VERRA_PROJECT_NAMES[i][2]
-          break
-        }
-      }
-    }
-
-    if (registry == 'PURO_EARTH') {
-      for (let i = 0; i < PURO_PROJECT_INFO.length; i++) {
-        if (projectID == PURO_PROJECT_INFO[i][0]) {
-          project.name = PURO_PROJECT_INFO[i][1]
-          project.country = PURO_PROJECT_INFO[i][2]
-          break
-        }
-      }
-    }
-
-    if (registry == 'ECO_REGISTRY') {
-      for (let i = 0; i < ECO_REGISTRY_PROJECT_INFO.length; i++) {
-        if (projectID == ECO_REGISTRY_PROJECT_INFO[i][0]) {
-          project.name = ECO_REGISTRY_PROJECT_INFO[i][1]
-          project.country = ECO_REGISTRY_PROJECT_INFO[i][2]
-          project.category = ECO_REGISTRY_PROJECT_INFO[i][4]
-          break
-        }
+    for (let i = 0; i < PROJECT_INFO.length; i++) {
```

Comment:
thinking back to when the subgraph hit memory limits in long loops in the past, maybe we could narrow the scope here and have registries keys within ProjectInfo so that we loop over PROJECT_INFO['registry']

Reply:
If it comes to that mùaybe we should use a map instead?
https://www.assemblyscript.org/stdlib/map.html